docs/endpoints/post-token-refresh.md

```diff
@@ -108,10 +111,11 @@ The following table lists the `status` property values and their HTTP status cod
 
 | Status | HTTP Status Code | Description |
 | :--- | :--- | :--- |
-| `success` | 200 | The request was successful. The response will be encrypted. |
+| `success` | 200 | The request was successful and a new UID2 token, with associated values, is returned in the response. The response is encrypted. |
 | `optout` | 200 | The user opted out. This status is returned only for authorized requests. |
 | `client_error` | 400 | The request had missing or invalid parameters.|
-| `invalid_token` | 400 | The request had an invalid identity token specified. This status is returned only for authorized requests. |
+| `invalid_token` | 400 | The UID2 token specified in the request was invalid. This status is returned only for authorized requests. |
+| `expired_token` | 400 | The UID2 token specified in the request was an expired token. |
```

Comment:
_The refresh token specified in the request was expired._

It's not the UID2 token but the refresh token (that contains the UID2 token) has expired

Reply:
Fixed. TY!